main.c

```diff
@@ -28,14 +26,10 @@ int	main(int ac, char **av, char **env)
 {
 	t_global	global;
 
-	if (ac == 2 && ft_strncmp(av[1], "testing", 7) == 0)
```

Comment:
could you leave this if clause - this triggers tester of my functions if we run minishell like that: `./minishell testing`